src/test/java/christmas/domain/dto/BenefitDtoTest.java

```diff
@@ -0,0 +1,30 @@
+package christmas.domain.dto;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import christmas.domain.Benefit;
+import org.junit.jupiter.api.Test;
+
+class BenefitDtoTest {
+    @Test
+    void BenefitDto는_Benefit을_문자열로_변환한다1() {
+        // given
+        Benefit benefit = new Benefit(true);
+        // when
+        BenefitDto dto = BenefitDto.fromBenefit(benefit);
+        String result = dto.getBenefit();
+        // then
+        assertThat(result).isEqualTo("샴페인 1개");
+    }
+
+    @Test
+    void BenefitDto는_Benefit을_문자열로_변환한다2() {
+        // given
+        Benefit benefit = new Benefit(false);
+        // when
+        BenefitDto dto = BenefitDto.fromBenefit(benefit);
+        String result = dto.getBenefit();
+        // then
+        assertThat(result).isEqualTo("없음");
+    }
+}
```

Comment:
```suggestion
class BenefitDtoTest {
    @Test
    void BenefitDto는_Benefit가_있으면_이름을_반환한다() {
        // given
        Benefit benefit = new Benefit(true);
        // when
        BenefitDto dto = BenefitDto.fromBenefit(benefit);
        String result = dto.getBenefit();
        // then
        assertThat(result).isEqualTo("샴페인 1개");
    }

    @Test
    void BenefitDto는_Benefit가_없으면_없음을_반환한다() {
        // given
        Benefit benefit = new Benefit(false);
        // when
        BenefitDto dto = BenefitDto.fromBenefit(benefit);
        String result = dto.getBenefit();
        // then
        assertThat(result).isEqualTo("없음");
    }
}
```
아니면 `Benefit가_없는경우` 이정도만 해도 될 것 같아요